### PR TITLE
Adding Jenkins file for PPC64LE JDK21

### PR DESCRIPTION
**Pull request by sanskruti1098**
Hi @oscerd to enable JDK21 jobs for power for camel- 4.x request to review and merge it

### CODE REVIEW

File: Jenkinsfile.ppc64le.jdk21 [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            }
+        }
+
+        stage('Code Quality Review') {
+            steps {
+                withCredentials([string(credentialsId: 'apache-camel-core', variable: 'SONAR_TOKEN')]) {
+                    sh "./mvnw $MAVEN_PARAMS -Dsonar.host.url=https://sonarcloud.io -Dsonar.java.experimental.batchModeSizeInKB=2048 -Dsonar.organization=apache -Dsonar.projectKey=apache_camel -Dsonar.branch.name=$BRANCH_NAME org.sonarsource.scanner.maven:sonar-maven-plugin:sonar"
+                }
+            }
+        }
```
Comment (orpiske) [inline Jenkinsfile.ppc64le.jdk21:72]: I don't think we need these on power.

**Review by orpiske — CHANGES_REQUESTED** [review]
There's some stuff we don't need there. I think it was incorrectly added in some of the current power jobs, but I'll clean them up.

**Comment by sanskruti1098** [comment]
Hi @orpiske @oscerd, should we remove whole 'Code Quality Review' stage?

**Comment by oscerd** [comment]
> Hi @orpiske @oscerd, should we remove whole 'Code Quality Review' stage?

yes, thanks.

**Comment by davsclaus** [comment]
What is the status of this PR ?

**Comment by sanskruti1098** [comment]
Hi @oscerd @orpiske I have done required changes request to review and merge it

File: Jenkinsfile.ppc64le.jdk21 [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+    }
+
+    post {
+        always {
+            emailext(
+                subject: '${DEFAULT_SUBJECT}',
+                body: '${DEFAULT_CONTENT}',
+                recipientProviders: [[$class: 'CulpritsRecipientProvider']]
```
Comment (oscerd) [inline Jenkinsfile.ppc64le.jdk21:87]: This should be DevelopersRecipientProvider, please

Comment (sanskruti1098) [reply Jenkinsfile.ppc64le.jdk21:87]: Ok

Comment (sanskruti1098) [reply Jenkinsfile.ppc64le.jdk21:87]: Ok

**Comment by sanskruti1098** [comment]
Hi @oscerd I have changed 'recipientProviders' request to review and merge it

**Review by orpiske — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

**Comment by davsclaus** [comment]
This should be in the main branch and not 4.3.x (non LTS)

**Comment by sanskruti1098** [comment]
Hi @oscerd can you please provide job link for JDK21 jobs for power for camel- 4.x

**Comment by oscerd** [comment]
I didn't create it

**Comment by oscerd** [comment]
I'll try to find time today

**Comment by sanskruti1098** [comment]
Ok, thank you..

**Comment by oscerd** [comment]
https://ci-builds.apache.org/job/Camel/job/Camel%20JDK21%20(ppcle64)/job/camel-4.3.x/

**Pull request merged**